applications/sink/router-sink/README.adoc

```diff
@@ -18,51 +18,25 @@ $$router.variables$$:: $$Variable bindings as a new line delimited string of nam
 $$router.variables-location$$:: $$The location of a properties file containing custom script variable bindings.$$ *($$Resource$$, default: `$$<none>$$`)*
 //end::configuration-properties[]
 
-NOTE: Since this is a dynamic router, destinations are created as needed; therefore, by default the `defaultOutputChannel`
-and `resolutionRequired` will only be used if the `Binder` has some problem binding to the destination.
+NOTE: This router sink is based on a `StreamBridge` API from Spring Cloud Stream, therefore destinations can be created as needed.
+In this case a `defaultOutputBinding` can only be reached if key is not included into `destinationMappings`.
```

Comment:
No, I don't think so. You see all of this information goes already after those properties. So, it assumes that you are already familiar with props to avoid confusion when you read this line.

Reply:
Not being very familiar w/ the router I just found it a bit confusing.  I made an attempt at trying to rewrite a bit, but was not successful. :) This is fine for now.